Dockerfile.test

```diff
@@ -0,0 +1,28 @@
+ARG RACKET_IMAGE
+
+FROM ${RACKET_IMAGE}
+
+RUN raco pkg install --auto --installation \
```

Comment:
I think we should just install `main-distribution-test` instead of most of these.

Reply:
Done, I think.